### PR TITLE
Support for airgap installs from the command line

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

**Comment by marccampbell** [comment]
This PR adds support for deploying an airgap install from the command line.

Given: a logged in registry at registry.somebigbank.com:

```
kubectl kots pull replicated://sentry-enterprise \
    --license-file ~/sentry-prod-license.yaml \
    --rewrite-images \
    --image-namespace sentry \
    --registry-endpoint registry.somebigbank.com 
    --shared-password password
```

This will result in all images from the deployment (application and kotsadm) retagged as registry.somebigbank.com/sentry/* and pushed. Also the midstream kustomization yaml will now contain:

```
apiVersion: kustomize.config.k8s.io/v1beta1
bases:
- ../../base
images:
- name: docker.io/bitnami/redis:4.0.11-debian-9
  newName: registry.somebigbank.com/sentry/redis
  newTag: 4.0.11-debian-9
- name: docker.io/kotsadm/kotsadm-api:alpha
  newName: registry.somebigbank.com/sentry/kotsadm-api
  newTag: alpha
- name: docker.io/kotsadm/kotsadm-operator:alpha
  newName: registry.somebigbank.com/sentry/kotsadm-operator
  newTag: alpha
- name: docker.io/kotsadm/kotsadm-web:alpha
  newName: registry.somebigbank.com/sentry/kotsadm-web
  newTag: alpha
- name: docker.io/library/postgres:10.7
  newName: registry.somebigbank.com/sentry/postgres
  newTag: "10.7"
- name: docker.io/library/postgres:9.6
  newName: registry.somebigbank.com/sentry/postgres
  newTag: "9.6"
- name: docker.io/library/sentry:9.1.1
  newName: registry.somebigbank.com/sentry/sentry
  newTag: 9.1.1
- name: docker.io/minio/mc:RELEASE.2019-07-17T22-13-42Z
  newName: registry.somebigbank.com/sentry/mc
  newTag: RELEASE.2019-07-17T22-13-42Z
- name: docker.io/minio/minio:RELEASE.2019-05-14T23-57-45Z
  newName: registry.somebigbank.com/sentry/minio
  newTag: RELEASE.2019-05-14T23-57-45Z
kind: Kustomization
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+This is intended te be the guide describing how to contribute to Kots and how to set up a dev environment.
+
+### Development dependencies
+
+To get started, you'll need Go 1.12 and a couple of OS packages:
+
+#### Debian / Ubuntu
+```
+sudo apt-get install libgpgme-dev
```
Comment (divolgin) [inline CONTRIBUTING.md]: don't need this anymore, right?

File: Makefile [hunk 2]
```diff
@@ -32,34 +32,35 @@ define LDFLAGS
 "
 endef
 
+BUILDTAGS = containers_image_ostree_stub exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
```
Comment (laverya) [inline Makefile]: Can we call this something like "SKOPEO_BUILDTAGS"?

File: CONTRIBUTING.md (hunk 1 above)
Comment (marccampbell) [reply CONTRIBUTING.md]: 👍  removed

File: deploy/.goreleaser.yml [hunk 3]
```diff
@@ -21,7 +21,7 @@ builds:
       -X github.com/replicatedhq/kots/pkg/version.gitSHA={{.FullCommit}}
       -X github.com/replicatedhq/kots/pkg/version.buildTime={{.Date}}
       -extldflags "-static"
-    flags: -tags netgo -installsuffix netgo
+    flags: -tags netgo netgo containers_image_ostree_stub exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp -installsuffix netgo
```
Comment (laverya) [inline deploy/.goreleaser.yml]: extra netgo here
```suggestion
    flags: -tags netgo containers_image_ostree_stub exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp -installsuffix netgo
```

File: Makefile (hunk 2 above)
Comment (divolgin) [reply Makefile]: I think `BUILDTAGS` is ok.  They do have global effect.

Comment (laverya) [reply Makefile]: Right, but we might have buildtags of our own eventually
Adding SKOPEO there would make it clear where they came from

**Pull request merged**